examples/kmeans/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "kmeans"
+version = "0.1.0"
+authors = ["m3ngyang <yangm3ng@gmail.com>"]
```

Comment:
We use a unified author declaration: `authors = ["MesaTEE Authors <developers@mesatee.org>"]`, and list our contributors in the other place.